utils/common-messages.ts

```diff
@@ -7,6 +7,14 @@ export const commonMessages = defineMessages({
     id: 'button.cancel',
     defaultMessage: 'Cancel',
   },
+  continueButton: {
+    id: 'button.continue',
+    defaultMessage: 'Continue',
+  },
+  createdAtLabel: {
+    id: 'label.created-at',
+    defaultMessage: 'Created {ago}',
+  },
```

Comment:
Here message content does not match the ID and property name.
```suggestion
  createdAgoLabel: {
    id: 'label.created-ago',
    defaultMessage: 'Created {ago}',
  },
```

Reply:
Yeah, that's better ^^.